### PR TITLE
Revert dev image Dockerfile to use internal workspace full base image

### PR DESCRIPTION
**Pull request by princerachit**
## Description

Revert the base image change for dev image so that it refers to the internal image generated from dazzlev2 images.
The `workspace-base-images:full` image already has tool-vnc as one of the chunks. This was changed with an assumption that we require the vnc image in order to run cypress tests.

## Related Issue(s)

None

## How to test

Following tests were performed. You can test the same cases. Create a new branch from this branch and then try to create a workspace in the corresponding preview env.

### Tests continue to work

image

### Commands run in preview env terminal

image

## Release Notes

```release-note
NONE
```

- [x] /werft with-clean-slate-deployment=

### CODE REVIEW

**Comment by roboquat** [comment]
@princerachit: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **princerachit** after the PR has been reviewed.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by kylos101 — APPROVED** [review]
/LGTM

Thank you, @princerachit, for the quick response.

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7742#pullrequestreview-859574661" title="LGTM">kylos101</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [kylos101]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7742#issuecomment-1018532250" title="Approved">kylos101</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/7742#issuecomment-1018532250" title="Approved">kylos101</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [kylos101]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
